.github/workflows/elixir-ci-v2.yml

```diff
@@ -0,0 +1,85 @@
+name: Elixir Quality CI
```

Comment:
Could you please rename this file elixir-library.yml and rename the workflow to "Elixir library CI" instead ?